src/modules/error/hocs/withPayloadProvider.js

```diff
@@ -94,7 +94,7 @@ const withPayloadProvider = <S: { dispatch: Dispatch<StoreActionType> }, R: {}>
             ? <ScrollView refreshControl={<RefreshControl refreshing />} contentContainerStyle={{ flexGrow: 0 }} />
             : null
         } else { // props.status === 'success'
-          return <ScrollView refreshControl={<RefreshControl onRefresh={this.refresh} refreshing={false} />}
+          return <ScrollView keyboardShouldPersistTaps='always' refreshControl={<RefreshControl onRefresh={this.refresh} refreshing={false} />}
```

Comment:
Just because I'm interested:
What impact does this have? On my android device, scrolling the scrollview didn't close the keyboard for me anyways...

Reply:
The inpact is, that you can now open the result with one tap. Before that, you could only close the keyboard with the first tap and had to tap again to open the result.

In order to close the keyboard while scrolling, we could add `keyboardDismissMode="on-drag"`.

Reply:
Ah okay cool, thanks for the hint!